### PR TITLE
Fix coverage collection

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR updates the tox config to run tests for dev packages only and adds no cover marker in the `__init__.py` modules

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -87,6 +87,33 @@ commands =
     python -m pip install --no-deps file://{toxinidir}/plugins/aea-test-autonomy
     aea test --cov --append packages -p no:hypothesispytest {posargs}
 
+[commands-packages-cov]
```
Comment (angrybayblade) [inline tox.ini:90]: Run tests for dev packages only, I've opened a ticket to do this at the command level on open-aea

File: packages/valory/protocols/tendermint/tests/test_tendermint.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        actual_envelope = Envelope.decode(envelope_bytes)
+        expected_envelope = envelope
+
+        assert expected_envelope.to == actual_envelope.to
+        assert expected_envelope.sender == actual_envelope.sender
+        assert (
+            expected_envelope.protocol_specification_id
+            == actual_envelope.protocol_specification_id
+        )
+        assert expected_envelope.message != actual_envelope.message
```
Comment (Karrenbelt) [inline packages/valory/protocols/tendermint/tests/test_tendermint.py:52]: yeah this is that oddity of the comparison returning False. May open an issue on open-aea side.

**Comment by Karrenbelt** [comment]
looking at the [code cov report](https://app.codecov.io/gh/valory-xyz/open-autonomy/pull/1676) one observes that 
1. several `__init__.py` files still are reported to be at 0% (instead of "No data available")
2. `<...>_pb2.py` files are being tracked, which are autogenerated from protobuf and should not be included

**Comment by DavidMinarsch** [comment]
> looking at the [code cov report](https://app.codecov.io/gh/valory-xyz/open-autonomy/pull/1676) one observes that
> 
> 1. several `__init__.py` files still are reported to be at 0% (instead of "No data available")
> 2. `<...>_pb2.py` files are being tracked, which are autogenerated from protobuf and should not be included
> 
> Why we want to add pragma: no cover to each individual `__init__.py` files instead of the specify the file pattern in `.coveragerc`?

I suggest you exclude the `_pb2.py` in the `setup.cfg` as we usually do.

Re `__init__.py` I prefer selective `no cover` as sometimes they do contain logic and a blanket ignore would be too intense.

File: tox.ini (hunk 1 above)
Comment (DavidMinarsch) [reply tox.ini:90]: Please reference tickets in comments for easier cross-ref

File: packages/valory/skills/register_termination_abci/__init__.py [hunk 3]
```diff
@@ -21,9 +21,9 @@
 This module contains a dummy ABCI skill in which the agents register and then pick up the termination signal.
 
 It is created only for testing purposes for the termination feature.
-"""
+"""  # pragma: nocover
 
-from aea.configurations.base import PublicId
+from aea.configurations.base import PublicId  # pragma: nocover
```
Comment (DavidMinarsch) [inline packages/valory/skills/register_termination_abci/__init__.py:26]: why two?

It should be enough to place it on line 25 for the whole file

Comment (angrybayblade) [reply packages/valory/skills/register_termination_abci/__init__.py:26]: It does not work with placing the no cover tag at the top, I tried it as the first thing

File: tox.ini (hunk 1 above)
Comment (angrybayblade) [reply tox.ini:90]: Done

**Review by Karrenbelt — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**